controllers/kubevirtmachine_controller.go

```diff
@@ -188,14 +189,17 @@ func (r *KubevirtMachineReconciler) reconcileNormal(ctx *context.MachineContext)
 		return ctrl.Result{}, nil
 	}
 
-	// Fetch SSH keys to be used for cluster nodes, and update bootstrap script cloud-init with public key
-	clusterNodeSshKeys := ssh.NewClusterNodeSshKeys(ctx.ClusterContext(), r.Client)
-	if persisted := clusterNodeSshKeys.IsPersistedToSecret(); !persisted {
-		ctx.Logger.Info("Waiting for ssh keys data secret to be created by KubevirtCluster controller...")
-		return ctrl.Result{RequeueAfter: 10 * time.Second}, nil
-	}
-	if err := clusterNodeSshKeys.FetchPersistedKeysFromSecret(); err != nil {
-		return ctrl.Result{}, errors.Wrap(err, "failed to fetch ssh keys for cluster nodes")
+	var clusterNodeSshKeys *ssh.ClusterNodeSshKeys
+	if !annotations.IsExternallyManaged(ctx.Cluster) {
```

Comment:
fyi... we need to be looking at the KubeVirtCluster and not the Cluster object here. This won't pass the e2e test i'm working on in #83.